src/Uno.UI/UI/Xaml/Controls/Image/Image.iOS.cs

```diff
@@ -263,6 +262,7 @@ private void UpdateContentMode(Stretch stretch)
 			}
 			else
 			{
+				ContentMode = UIViewContentMode.ScaleToFill;
```

Comment:
Why?

Reply:
Required by the new way the layout is done for images.

Reply:
Why? 😁

Reply:
The image need to take all the calculated space.